d3a_api_client/redis_market.py

```diff
@@ -11,7 +12,7 @@
 
 class RedisMarketClient:
     def __init__(self, area_id, redis_url='redis://localhost:6379'):
-        self.area_id = area_id
+        self.area_id = slugify(area_id, to_lower=True)
```

Comment:
Can you please check whether this is also needed in `RedisClient`?
Also, why not changing this on the d3a side and skip slugifying the name here?

Reply:
I checked and this is not needed for `RedisClient ` as devices don't slugify their channel prefix whereas area market do, you are right that it could be done on d3a side but it would be a rather time consuming refactoring (may be not so important either) that is to adapt not just `d3a` but also `d3a-web` too, so I took the the easier approach to simply slugify on d3a-api-client side.

Reply:
Actually, I don't see a point in the d3a-web where the channel is sluggified. But yeah, you are right. It is too much work for this side task. But IMO we should be consistent on all parts. So If you agree, please add a refactoring story that either removes the sluggification for the channel names or add it to all (market AND device) .